docs/release-notes/8.0.32-24.md

```diff
@@ -0,0 +1,70 @@
+# Percona Server for MySQL 8.0.32-24 (2023-03-12)
+
+<style>
+    table {
+        width=50%’;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | {{ release_date }} |
+|--- | --- |
+| Install instructions | [Install Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html) |
+
+[Percona Server for MySQL 8.0.32-24](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.32 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-32.html)
+in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open
+source drop-in replacement for any MySQL database. It provides superior
+performance, scalability, and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to MySQL server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility. 
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Percona decided to revert the following MySQL bug fix:
+
+The data and the GTIDs backed up by mysqldump were inconsistent when the options --single-transaction and --set-gtid-purged=ON were both used. It was because in between the transaction started by mysqldump and the fetching of GTID_EXECUTED, GTIDs on the server could have increased already. With this fixed, a FLUSH TABLES WITH READ LOCK is performed before the fetching of GTID_EXECUTED to ensure its value is consistent with the snapshot taken by mysqldump.
+
```

Comment:
Also in the highlights we need to explicitly state that after oracle fixed
```
InnoDB: Undetectable problems after upgrade from 8.0.28, crash and corruption.

Any new row inserted after upgrade will have all columns added with ALGORITHM=INSTANT materialized and have version=0.

In the new implementation, a column added with ALGORITHM=INSTANT will fail if the maximum possible size of a row exceeds the row size limit. So new rows with materialized ALGORITHM=INSTANT columns will always be within row size limit. (Bug #34558510)
```
,
```
A specific column added after a drop using the INSTANT algorithm could cause a data error and a server exit. (Bug #34122122)
```
, and
```
InnoDB: After a column added with ALGORITHM=INSTANT, an online rebuild DDL no longer crashes.

Thank you Qingda Hu for reporting this bug. (Bug #33788578, Bug #106279)
```
we reverted our fix https://jira.percona.com/browse/PS-8292 and `INSTANT` is again the default ALTER TABLE algorithm for supported operations.

Reply:
Added.